lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -221,4 +224,12 @@ module.exports = {
 
     return auth;
   },
+
+  validateAlbHealthCheck(event) {
+    const eventHealthCheck = event.alb.healthCheck;
+    if (_.isObject(eventHealthCheck)) {
+      return Object.assign(eventHealthCheck, { enabled: true });
+    }
+    return { enabled: eventHealthCheck };
```

Comment:
Let's do simply `{ enabled: true }` (to avoid side effect is someone used non-boolean truthy value)